tutor/src/models/student-tasks/step.js

```diff
@@ -24,7 +24,9 @@ class StudentTaskPlaceHolderStep extends TaskStepContent { }
 
 class StudentTaskReadingStep extends TaskStepContent {
   @observable title;
-  @lazyGetter chapterSection = new ChapterSection(this.chapter_section);
+  @lazyGetter chapterSection = new ChapterSection(
+    this.baked_chapter_section || this.chapter_section
```

Comment:
Seems odd that this file and reference-book/page.js accomplish the same thing in slightly different ways.

Reply:
yes it is a bit cumbersome.  the data is quite a bit different for the two though so it's kinda hard to merge them